packages/@aws-cdk/aws-codepipeline/test/test.pipeline.ts

```diff
@@ -518,7 +531,7 @@ export = {
       test.done();
     },
   },
-};
+} as any;
```

Comment:
I'm suspicious why this one is necessary. I'd remove it as it "disables" type checking in ways that can hide other issues (see below).

Last I saw this done, it was motivated by the TypeScript compiler being confused by a genuine coding error somewhere else (it was a use of `this` - instead of some `testStack` - as the first argument of a `Construct` class instantiation caused `tsc` to assume the whole test group to be a `Construct`- which is obviously not the case - and that caused a very confusing error message to be issued).

Reply:
I will send update PR. I don't remember this situation exactly, TS was complaining that test attribute `'test name'` can't be is not some other class (maybe `Construct`). 

It was very wired. I think it was due to import of some other files, which I removed after clean build (I think some circular dependencies).